packages/core/package.json

```diff
@@ -15,6 +15,9 @@
     "vscode-languageserver-protocol": "^3.17.2",
     "vscode-ws-jsonrpc": "^2.0.1"
   },
+  "devDependencies": {
+    "deepmerge-ts": "^4.2.2"
```

Comment:
welcome back :smile:

Reply:
was never gone, and only as a dev dependency (like test dependency in maven)

Reply:
ahh see, thanks for clarifying